src/components/timeline/timeline.component.tsx

```diff
@@ -26,24 +26,26 @@ import { useMyriadUser } from 'src/hooks/use-myriad-users.hooks';
 import { usePolkadotApi } from 'src/hooks/use-polkadot-api.hook';
 import { useTimelineHook } from 'src/hooks/use-timeline.hook';
 import { Post } from 'src/interfaces/post';
+import { Token } from 'src/interfaces/token';
 
 type TimelineProps = {
   isAnonymous: boolean;
+  availableTokens: Token[];
 };
 
-const Timeline: React.FC<TimelineProps> = ({ isAnonymous }) => {
+const Timeline: React.FC<TimelineProps> = ({ isAnonymous, availableTokens }) => {

```

Comment:
We haven't moved to Redux, we're still using Context. So I need to pass around availableTokens from parents to child (to avoid using hooks at too many places).
We have planned to migrate from Context to Redux but haven't had the time yet.

Reply:
tapi context ada global store juga kan? Why can't you just keep accessing the `availableTokens` from the context?